devicetypes/smartthings/aeon-multisensor.src/aeon-multisensor.groovy

```diff
@@ -236,4 +263,20 @@ private setConfigured(configure) {
 
 private isConfigured() {
 	getDataValue("configured") == "true"
+}
+
+private sendCheckInterval() {
+	def checkInterval
+	if(zwaveInfo.mfr == "027A") checkInterval = 8 * 60 * 60 + 2 * 60
```

Comment:
The formula going forward for Z-Wave sleepy devices should be `(2 * Wake up Interval ) + ( 10 minutes) `

Reply:
I would suggest to set the `def wakeupInterval` and then before sending the check interval do ` checkInterval = 2 * checkInterval + (10 * 60)`

Reply:
Wasn't 4h a default wakeUpInterval time value, or I have missed something in the meantime?

Reply:
It does not appear that we are configuring the Wake up interval to be something other than 4 hours, I thought we were, sorry about the confusion. Having said that, we should have the same checkInterval for all devices using this DTH. 

We should use this formula going forward for all Z-Wave sleepy devices `checkInterval = 2 * checkInterval + (10 * 60)`.